### ISSUE TITLE
Unable to load template 'file:layout-ajax.tpl

### ISSUE THREAD

**Issue by Juiliard** [PrestaShop/PrestaShop#35225]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

After Upgrading from PS 1.7.8.11 to PS 8.1.3 the following error shown in Symfony: 
302 Redirect from  @n/a ([8270c5](https://XXX/index.php/_profiler/8270c5?panel=request)),
Forwarded to [ErrorController](https://XXX/adminXXX/index.php/_profiler/open?file=vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Controller/ErrorController.php&line=26#line26) ([d57993](https://XXX/adminXXX/index.php/_profiler/d57993)) Method: GET, HTTP Status: 500
An exception has been thrown during the rendering of a template ("Unable to load template 'file:layout-ajax.tpl'").
Please advise how to fix it.
<img width="1100" alt="Screenshot 2024-01-30 at 16 57 24" src="https://github.com/PrestaShop/PrestaShop/assets/58143423/7982bd9d-6876-4535-a690-9d467f4be73a">

### Expected behavior

no errors in Symfony to be shown

### Steps to reproduce

PS 1.7.6.5 Updated to PS 1.7.8.11
PS 1.7.8.11  Upgraded to PS 8.1.3
Activate Debug mode
check for Ayax requests  ( red icon with two errows at the bottom of the screen)
open the request with status 500

### PrestaShop version(s) where the bug happened

8.1.3

### PHP version(s) where the bug happened

8.1.18

**Comment by florine2623** [comment]
Hello @Juiliard ,

I upgraded from 1.7.8.11 to 8.1.3 successfully using module 1-Click Upgrade v5.0.0.
<img width="378" alt="Screenshot 2024-01-31 at 10 07 12" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/c7a9aff3-e38c-42c6-a2da-fdd67d055b56">

I didn't have issues with the performance page.

https://github.com/PrestaShop/PrestaShop/assets/16019289/4d291ac8-05ee-4423-b105-a8ed167aea20

Did you have an issue upgrading from 1.7.6.5 to 1.7.8.11 ?

**Comment by Juiliard** [comment]
Thanks for checking @florine2623 .
 I didn’t see any error, but actually I did not check symfonie when upgraded to 1.7.8.11.
Would you know what may be the cause of this error anyhow?

**Comment by florine2623** [comment]
Hello @Juiliard ,

I wouldn't be able to tell you without seeing your shop.

We use GitHub issues only to discuss about bugs and new features in the PrestaShop project. If you need support, you can find resources to help you on this page: [Get help with PrestaShop](https://www.prestashop-project.org/support/).

Alternatively, you can also ask for help in the [community forums](https://www.prestashop.com/forums/) or in the [public Slack channel](https://github.com/PrestaShop/open-source/tree/master/slack).

Thank you
